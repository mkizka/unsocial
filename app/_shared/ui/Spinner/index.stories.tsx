import type { Meta, StoryObj } from "@storybook/react";

import { Spinner } from ".";

const meta: Meta<typeof Spinner> = {
  component: Spinner,
  args: {
    className: "w-8 h-8 text-secondary",
  },
};
export default meta;

type Story = StoryObj<typeof Spinner>;

export const Default: Story = {};
