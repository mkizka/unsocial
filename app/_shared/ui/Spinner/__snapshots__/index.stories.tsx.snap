// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`_shared/ui/Spinner Default smoke-test 1`] = `
<svg class="animate-spin w-8 h-8 text-secondary"
     xmlns="http://www.w3.org/2000/svg"
     fill="none"
     viewbox="0 0 24 24"
>
  <circle class="opacity-25"
          cx="12"
          cy="12"
          r="10"
          stroke="currentColor"
          stroke-width="4"
  >
  </circle>
  <path class="opacity-75"
        fill="currentColor"
        d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"
  >
  </path>
</svg>
`;
